### PR TITLE
#DONOTMERGE Update Hokusai to v0.5.8 and add new SSL certs with variable interpolation

### PR DESCRIPTION
**Pull request by izakp**
- Update Hokusai to v0.5.8 and add hokusai-vars.yml to ./hokusai/config.yml

- Use artsyNetWildcardSSLCert variable in ./hokusai/staging.yml and ./hokusai/production.yml

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>The staging version of Metaphysics has breaking changes, this might not affect your build, but could mean your queries don't work the way you expect. See below for breaking changes:</td>
    </tr>
  </tbody>
</table>


- :warning: Field 'isTargetSupply' was removed from object type 'Artist'
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 229b17ced4dbb1e025b679da135fd9177bd2b2bd
</p>

File: hokusai/config.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 project-name: force
 pre-deploy: yarn publish-assets
 post-deploy: .circleci/deploy_event.sh
-hokusai-required-version: ">=0.5.4"
+hokusai-required-version: ">=0.5.8"
```
Comment (eessex) [inline hokusai/config.yml:4]: I thought we were recommending a loose version here, `~=0.5` is what we had most apps move to during hokusai checklist update processes. My understanding was that the orb should control the specific version.

Comment (izakp) [reply hokusai/config.yml:4]: Yes but this change is not backward compatible... devs would get errors if running on an older Hokusai version

Comment (izakp) [reply hokusai/config.yml:4]: The orb will install the latest 0.5.x version

**Pull request closed without merging**